server/database/config/build.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  sequelize,
+  AdminProduct as adminProductsModel,
+  Pharmacy as pharmacyModel,
+  Product as productsModel,
+  ProductPharmacy as productsPharmaciesModel,
+  ProductsRequest as productsRequestsModel,
+  SalesHistory as salesHistoryModel,
+} from '../../models';
+
+import {
+  adminProducts,
+  pharmacies,
+  products,
+  productsPharmacies,
+  productsRequests,
+  salesHistory,
+} from '../seeds';
+
+// console.log(sequelize);
+const buildDB = async () => {
```

Comment:
Good Job, I need you to create two functions, one to build the tables and another to insert your fake data.
And add to scripts to run them.